examples/async/composite_agg.py

```diff
@@ -52,13 +53,13 @@ async def run_search(**kwargs: Any) -> Response:
         return await s.execute()
 
     response = await run_search()
-    while response.aggregations.comp.buckets:
-        for b in response.aggregations.comp.buckets:
-            yield b
-        if "after_key" in response.aggregations.comp:
-            after = response.aggregations.comp.after_key
+    while response.aggregations["comp"].buckets:
```

Comment:
Switching from `.comp` to `["comb"]` feels like a net negative, but if that helps mypy type correctly (and raise an error on a typo) then that's OK!

Reply:
I'm not sure I agree. To me it feels like using the dict syntax makes it clear that this is a dynamic data item that is application specific. But maybe I convinced myself this is better.